solutions/usaco-940.mdx

```diff
@@ -87,6 +87,61 @@ public class Factory {
 ```
 
 </JavaSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
+using namespace std;
+
+void setIO(string name=""){ // FastIO See General -> Input/Output
+    ios_base::sync_with_stdio(0); cin.tie(0);
+    if((int)name.size() > 0){
+        freopen((name+".in").c_str(), "r", stdin);
+        freopen((name+".out").c_str(), "w", stdout);
+    }
+}
+
+int main(){
+    setIO("factory");
+    int N;
+    cin >> N;
+    int outgoing[N] = {0};
+    // N - 1 walkways
+    for(int i = 0; i < N -1; i++){
+        int a,b;
```

Comment:
spaces after control keywords and operators
```
for (int i = 0; i < n; i++) {
    int a, b;
}
```

Reply:
Done